ec/acc/deployment_basic_defaults_test.go

```diff
@@ -133,6 +131,42 @@ func TestAccDeployment_basic_defaults(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "enterprise_search.#", "0"),
 				),
 			},
+		},
+	})
+}
+
+func TestAccDeployment_basic_defaults_hw(t *testing.T) {
+	resName := "ec_deployment.defaults"
+	randomName := prefix + acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	startCfg := "testdata/deployment_basic_defaults_3.tf"
+	secondCfg := "testdata/deployment_basic_defaults_4.tf"
```

Comment:
if we're splitting the tests and using different prefixes, then it would be goo to name the `.tf` files the same, right?